schema/html5/sectional.rnc

```diff
@@ -129,6 +129,7 @@
 			|	h5.elem
 			|	h6.elem
 			)
+			&	p.elem*
 			&	common.elem.script-supporting*
 		)+
 
```

Comment:
h1-h6 element must be only one in hgroup element.

https://html.spec.whatwg.org/multipage/sections.html#the-hgroup-element
> Content model:
>   Zero or more p elements, followed by **one** h1, h2, h3, h4, h5, or h6 element, followed by zero or more p elements, optionally intermixed with script-supporting elements.

Reply:
Thank you for noticing. New commit made hgroup with no heading to show following message:
![image](https://user-images.githubusercontent.com/100634371/199063794-7d1c67f4-4089-40d9-bbf4-3b58669b2cee.png)
While hgroup with more than one heading now shows:
![image](https://user-images.githubusercontent.com/100634371/199063872-85a5034e-c89d-44f7-a599-9e2d63ff270a.png)

Error message saying "one or more" children are required, is a bit misleading. I have created separate issue in jing-trang about it: [#276](https://github.com/relaxng/jing-trang/issues/276).